src/icloudpd/base.py

```diff
@@ -488,7 +488,7 @@ def report_version(ctx: click.Context, _param: click.Parameter, value: bool) ->
     is_flag=True,
 )
 @click.option(
-    "--keep-recent-days",
+    "--keep-icloud-recent-days",
     help="When using --delete-after-download, do not delete photos newer than this many days",
     type=click.IntRange(0),
     default=0,
```

Comment:
We do not need default equivalent of wiping the whole iCloud collection. Without default the value of the param is None.

Reply:
Updated default. Initial check below was already checking for `None`. Good call!